rkt/rkt.mk

```diff
@@ -32,3 +32,11 @@ include makelib/build_go_bin.mk
 $(call undefine-namespaces,LOCAL)
 # RKT_STAMP deliberately not cleared
 # RKT_BINARY deliberately not cleared
+
+manpages:
```

Comment:
> It would make the compilation quite slow if we run `make manpages/bash-completion` at each `make` though... On my laptop:
>
> - time make manpages: 25 seconds
- time make bash-completion: 24 seconds
- total: 49 seconds

I wanted it to be a part of `make all`, because it is about building everything. Possibly this could be controlled with `--enable-man` and `--enable-bash-completion` flags which could be disabled by default.

> > Pretty printers could be used there.
> > No big deal, since the output is rather short.
>
> How to do that?

```
manpages:
	$(VQ) \ <- this expands to @ when pretty printing (V=0,1,2) or to nothing for raw output (V=3)
	set -e;
	$(call vb,vt,GEN,manpages) \ <- this will print (GEN     manpages for V=1,2, nothing otherwise)
	ls rkt/*.go | blah blah blah
```